common-components/proto/api/type/money.proto

```diff
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package api.type;
```

Comment:
proto package name doesn't match go / java package name.

Reply:
Should it?  Go and Java seem to want something that corresponds to a domain, which implies that it has to contain aip.dev.

But these common components themselves are part of a package _for_ APIs (and _by_ AIPs, if you like)...not a package _for_ AIPs.

I don't know if the rationale above is correct, but it's how I came up with what's currently there.

Reply:
Updated the proto package.